addons/mail/static/tests/chatter_tests.js

```diff
@@ -2,7 +2,7 @@ odoo.define('mail.chatter_tests', function (require) {
 "use strict";
 
 var mailTestUtils = require('mail.testUtils');
-
+var AttachmentBox = require('mail.AttachmentBox');
```

Comment:
alphabetical order of grouped modules, so `mail.AttachmentBox` should be before `mail.testUtils`